### ISSUE TITLE
Stroke order error in 兜

### ISSUE THREAD

**Issue by cxy163** [skishore/makemeahanzi#57]
Pls check it.

**Comment by parsimonhi** [comment]
Seems the correct order in simplified Chinese. But stroke order in Japanese and traditional Chinese (at least in Taiwan) is different: 白 component is drawn first.

**Comment by cxy163** [comment]
The data in median.bin is right, but the svg file is wrong

**Comment by cxy163** [comment]
Same problem in 翰

**Comment by cxy163** [comment]
Yes, 白 component should be drawn first. But the data now is not like this.

**Comment by parsimonhi** [comment]
In simplified Chinese, 白 component is not drawn first!

**Comment by cxy163** [comment]
FYI, the latest standard in simplified Chinese, 白 component is drawn first

**Comment by parsimonhi** [comment]
Very interesting! Where can we find this latest standard?

**Comment by cxy163** [comment]
Can’t find electronic version, but only paper book.
https://item.jd.com/28695035898.html

Sent from Mail for Windows 10

**Comment by parsimonhi** [comment]
Many thanks! I think I'm going to have a new book on my shelves in not a long time! :-)
